i18n/locales/source/course_content/2021/coursef-2021.json

```diff
@@ -2788,7 +2794,7 @@
+    "long_instructions": "# Chance\n\n<div style=\"display:inline-block\">\n\n_Right now, it's impossible for monsters to give or get the virus if either of them is wearing a mask. This sounds great, but it's not really that simple. We should add a \"chance\" (or probability) for the virus to spread each time any two monsters touch. We can model the chance of getting the virus off of these notes._\n\n</div>\n\n<details open><summary><em><strong>Notes:</strong>How face masks affect the Monster Town outbreak</em></summary>\n\n<div style=\"text-align:center;\">\n<div style=\"background-color:indianred; text-align:center; border-style: solid; display:inline-block; margin:0 auto;\">\n<img src=\"https://images.code.org/01291b1e958448f65abb46b0d135b916-image-1603206161098.png\" style=\"width:55%\">\n\n<div style=\"background-color:white; text-align:center; border-style: solid; margin:0 15px;\">\n<br>\n\n_Benefits of Face Masks_\n\n| Healthy |  Sick |     Chance    |\n| :-----: | :---: | :-----------: |\n|  [][6]  | [][7] | **Very High** |\n|  [][4]  | [][5] |    **High**   |\n|  [][2]  | [][3] |    **Low**    |\n|  [][0]  | [][1] |  **Very Low** |\n\n<br>\n</div>\n<br>\n</div>\n</div>\n\n</details>\n\n## Do This\n\n1.  Add this **logic** block under the `when touches` event and set it to a very high chance:\n\n[blockly block][8]\n\n2.  Add more events that include monsters wearing masks and give each a different percent chance (probability). Click the lightbulb for a hint if you are not sure which events are needed.\n\n3.  Press **Finish** when you are ready to move on.",
```

Comment:
Looks like the redaction indexes changed in this string. Could someone verify this is expected?

Reply:
i believe this is part of @KatieShipley 's change

Reply:
That is the line with the probability table that had the monsters in the wrong order (which is what my one-off script fixes), so this might actually fix that problem. The safe bet is probably to exclude this from this sync and do it after HoC. Or we could push it through anyway; if it messes up the table I already have a script ready to fix it, so it's probably not a big deal either way.

Reply:
I'm okay with pushing it through.

Reply:
On Crowdin, it looks like this string is already using the version of the source string we are changing it to? https://crowdin.com/translate/codeorg/all/enus-es?filter=basic&value=3#q=%5B%5D%5B0%5D%20%20%7C%20%5B%5D%5B1%5D%20%7C%20**Very%20High**%20